lib/monotime.h

```diff
@@ -53,6 +53,17 @@ static inline time_t monotime(struct timeval *tvo)
 	return ts.tv_sec;
 }
 
+static inline time_t monotime_raw(struct timeval *tvo)
```

Comment:
What is this?  What does this have to do with this code?

Reply:
Currently every-time routes are updated it is updated with timestamp. When BGP goes for restart we note down the time. Any route less than this restart time post stale timer expiry is considered stale and will be removed. Timestamp used here was monotonic which could change with NTP so we used monotonic_raw. Unfortunately few platforms are not supporting the as per CI-CD results I saw. I will take care of that.

Reply:
Please fix your commits to actually only have what you are actually changing in it.  If you look carefully at this particular commit, the commit message clearly states you are adding a zapi message.  But we have a new added function in here that is not in any way shape fashion or form related to the commit message or what the rest of the commit is doing.  So when I saw it in isolation and what the commit message was for I was more than a bit confused.

Reply:
My bad I am addressing this. As per first comment I am removing use of monotime_raw so I will remove this function. Thanks for pointing it out.